eclass/acct-user.eclass

```diff
@@ -426,27 +331,51 @@ acct-user_src_install() {
 acct-user_pkg_preinst() {
 	debug-print-function ${FUNCNAME} "${@}"
 
-	# check if user already exists
-	_ACCT_USER_ALREADY_EXISTS=
-	if [[ -n $(egetent passwd "${ACCT_USER_NAME}") ]]; then
-		_ACCT_USER_ALREADY_EXISTS=1
```

Comment:
You didn't remove the respective eclassdoc. Don't ask me why I've documented it though.